addOns/openapi/src/test/java/org/zaproxy/zap/extension/openapi/generators/HeadersGeneratorUnitTest.java

```diff
@@ -391,9 +391,9 @@ void shouldGenerateAllSupportedHeaders() {
                 headers,
                 contains(
                         header("accept", "text/plain"),
-                        header("content-type", "application/json"),
                         header(headerName, headerValue),
-                        header("cookie", cookieName + "=" + cookieValue)));
+                        header("cookie", cookieName + "=" + cookieValue),
+                        header("content-type", "application/json")));
```

Comment:
Changing the order of operations in `generate` meant a change tot he order of the headers in the list.

Reply:
Maybe we should have a deny list for the headers we generate ourselves?